app/assets/stylesheets/custom.css

```diff
@@ -1,3 +1,11 @@
 h1 {
   text-align: center;
 }
+
+.items {
+  background-color: #d3d3d3;
+  padding: 10px;
+  margin: 20px;
+  border-radius: 25px;
+  text-align: center;
+}
```

Comment:
No I mean like, so that it doesn't look like this:
![screen shot 2015-03-14 at 12 28 08 pm](https://cloud.githubusercontent.com/assets/3409645/6652941/9f1f8d68-ca45-11e4-999a-f1b715e9fe48.png)